modules/lx-instance/watchmaker-lx-instance.template.cfn.yaml

```diff
@@ -0,0 +1,749 @@
+                  ${local_UseAdminGroups}
+                  ${local_UseAdminUsers}
+                -
+                  local_UseWamConfig: !If [UseWamConfig, !Sub '--config ${WatchmakerConfig}', '']
+                  local_UseEnvironment: !If [UseEnvironment, !Sub '--env ${WatchmakerEnvironment}', '']
+                  local_UseOUPath: !If [UseOuPath, !Sub '--ou-path ${WatchmakerOuPath}', '']
+                  local_UseComputerName: !If [UseComputerName, !Sub '--computer-name ${WatchmakerComputerName}', '']
+                  local_UseAdminGroups: !If [UseAdminGroups, !Sub '--admin-groups ${WatchmakerAdminGroups}', '']
+                  local_UseAdminUsers: !If [UseAdminUsers, !Sub '--admin-users ${WatchmakerAdminUsers}', '']
+      ToggleCfnInitUpdate: !Ref ToggleCfnInitUpdate
+    Properties:
+      BlockDeviceMappings:
+        - DeviceName: !Sub 
+            - '/dev/${local_Distro2RootDevice}'
+            - local_Distro2RootDevice: !FindInMap [Distro2RootDevice, !Ref AmiDistro, DeviceName]
+          Ebs:
+            DeleteOnTermination: true
+            VolumeType: gp2
+        - !If 
+          - CreateAppVolume
+          - DeviceName: /dev/xvdf
+            Ebs:
+              DeleteOnTermination: true
+              VolumeSize: !Ref AppVolumeSize
+              VolumeType: !Ref AppVolumeType
+          - !Ref 'AWS::NoValue'
+      IamInstanceProfile: !If 
+        - AssignInstanceRole
+        - !Ref InstanceRole
+        - !Ref 'AWS::NoValue'
+      ImageId: !Ref AmiId
+      InstanceType: !Ref InstanceType
+      KeyName: !Ref KeyPairName
+      NetworkInterfaces:
+        - AssociatePublicIpAddress: !If 
+            - AssignPublicIp
+            - true
+            - false
+          DeviceIndex: '0'
+          GroupSet: !Ref SecurityGroupIds
+          PrivateIpAddress: !If 
+            - AssignStaticPrivateIp
+            - !Ref PrivateIp
+            - !Ref 'AWS::NoValue'
+          SubnetId: !Ref SubnetId
+      Tags:
+        - Key: Name
+          Value: !Sub '${AWS::StackName}'
+      UserData: 
+        !Base64
+          Fn::Sub:
+            - |
+              Content-Type: multipart/mixed; boundary="===============3585321300151562773=="
+              MIME-Version: 1.0
+              --===============3585321300151562773==
+              Content-Type: text/cloud-config; charset="us-ascii"
+              MIME-Version: 1.0
+              Content-Transfer-Encoding: 7bit
+              Content-Disposition: attachment; filename="cloud.cfg"
+
+              #cloud-config
+              ${local_CreateAppVolume}
+
+              --===============3585321300151562773==
+              Content-Type: text/x-shellscript; charset="us-ascii"
+              MIME-Version: 1.0
+              Content-Transfer-Encoding: 7bit
+              Content-Disposition: attachment; filename="script.sh"
+
+              #!/bin/bash -xe
+
+              # Export AWS ENVs
+              test -r /etc/aws/models/endpoints.json && export
```

Comment:
see same suggestions as for the autoscale template

Reply:
fixed